var/spack/repos/builtin/packages/py-xattr/package.py

```diff
@@ -3,17 +3,16 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
-
 
 class PyXattr(PythonPackage):
     """A python interface to access extended file attributes,
-        sans libattr dependency"""
+    sans libattr dependency.
+    """
 
     homepage = "http://pyxattr.k1024.org/"
-    git      = "https://github.com/fwang2/pyxattr.git"
+    git      = "https://github.com/iustin/pyxattr.git"
```

Comment:
Is this package the same as https://pypi.org/project/xattr/? In that case, we could download tarballs straight from PyPI.

Reply:
Done